webapp/components/link.tsx

```diff
@@ -22,14 +23,31 @@ const getQuery = function (q: UrlObject['query']): Record<string, string> {
   return query
 }
 
-// When using link to navigate between pages, we need to persist the query string
-// for network type; otherwise, when navigating, users would switch between mainnet and testnet
-// due to the loss of this value
+// When using link to navigate between pages, we only persist the networkType
+// query parameter when it differs from the default network type. This keeps
+// the URL clean for the default network while still allowing navigation
+// between mainnet and testnet when explicitly selected.
 export const Link = function (props: ComponentProps<typeof BaseLink>) {
   const [networkType] = useNetworkType()
-  const href =
-    typeof props.href === 'string'
-      ? { pathname: props.href, query: { networkType } }
-      : { ...props.href, query: { ...getQuery(props.href.query), networkType } }
+  const defaultNetworkType = featureFlags.mainnetEnabled ? 'mainnet' : 'testnet'
```

Comment:
This is also defined in [useNetworkType](https://github.com/hemilabs/ui-monorepo/blob/852e37a6910fb9d16915e371843b8bdabec13703/webapp/hooks/useNetworkType.ts#L14).

In that file, I'd move that ternary outside of the hook (but inside the same file), and export it as `defaultNetwork`. Then, I'd import it here. 
This way, when we remove the featureFlag, we won't have to rewrite this again to import the default network